web/themes/custom/recipe_theme/src/sass/base/_elements.scss

```diff
@@ -1,30 +1,76 @@
-// Headings
+// Headings, Text
 // -----------------------------------------------------------------------------
 h1,
 h2,
 h3,
 h4,
 h5,
-h6 {
+h6,
+p {
+  font-family: $font-family-base;
+  font-style: normal;
 }
 
-// Text
-// -----------------------------------------------------------------------------
+h1,
+h2,
+h3,
+h5 {
+  text-align: center;
+  font-weight: bold;
+}
+
+h1 {
+  text-transform: uppercase;
+}
+
+h4,
+h6,
 p {
+  font-weight: normal;
 }
 
 // Links
 // -----------------------------------------------------------------------------
 a {
+  color: $orange;
+  font-family: $font-family-monospace;
+  font-weight: 600;
   @include hover {
+    color: $orange;
+    text-decoration-line: underline;
   }
 }
 
+a.navbar-brand {
```

Comment:
Please use nesting and the [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector). E.g.

```sass
a {
  &.class-1 {
    ...
  }
  &.class-2 {
    ...
  }
}
```

Reply:
Ok, corrected.